deepchem/molnet/load_function/material_datasets.py

```diff
@@ -0,0 +1,359 @@
+"""
+Datasets for inorganic crystal structures.
+"""
+import os
+import logging
+import deepchem
+from deepchem.feat import Featurizer
+from deepchem.trans import Transformer
+from deepchem.splits.splitters import Splitter
+from deepchem.molnet.defaults import get_defaults
+
+from typing import List, Tuple, Dict, Optional
+
+logger = logging.getLogger(__name__)
+
+# TODO: Change URLs
+DEFAULT_DIR = deepchem.utils.get_data_dir()
+BANDGAP_URL = 'http://deepchem.io.s3-website-us-west-1.amazonaws.com/datasets/expt_gap.tar.gz'
+PEROVSKITE_URL = 'http://deepchem.io.s3-website-us-west-1.amazonaws.com/datasets/perovskite.tar.gz'
+
+# dict of accepted featurizers for this dataset
+# modify the returned dicts for your dataset
+DEFAULT_FEATURIZERS = get_defaults("feat")
+
+# Names of supported featurizers
+featurizers = [
+    'ElementPropertyFingerprint', 'SineCoulombMatrix',
+    'StructureGraphFeaturizer'
+]
+DEFAULT_FEATURIZERS = {k: DEFAULT_FEATURIZERS[k] for k in featurizers}
+
+# dict of accepted transformers
+DEFAULT_TRANSFORMERS = get_defaults("trans")
+
+# dict of accepted splitters
+DEFAULT_SPLITTERS = get_defaults("splits")
+
+# names of supported splitters
+splitters = ['RandomSplitter']
+DEFAULT_SPLITTERS = {k: DEFAULT_SPLITTERS[k] for k in splitters}
+
+
+def load_bandgap(
+    featurizer: Featurizer = DEFAULT_FEATURIZERS['ElementPropertyFingerprint'],
+    transformers: List[Transformer] = [
+        DEFAULT_TRANSFORMERS['NormalizationTransformer']
+    ],
+    splitter: Splitter = DEFAULT_SPLITTERS['RandomSplitter'],
+    reload: bool = True,
+    data_dir: Optional[str] = None,
+    save_dir: Optional[str] = None,
+    featurizer_kwargs: Dict[str, object] = {'data_source': 'matminer'},
+    splitter_kwargs: Dict[str, object] = {
+        'frac_train': 0.8,
+        'frac_valid': 0.1,
+        'frac_test': 0.1
+    },
+    transformer_kwargs: Dict[str, Dict[str, object]] = {
+        'NormalizationTransformer': {
+            'transform_X': True
+        }
+    },
+    **kwargs) -> Tuple[List, Tuple, List]:
+  """Load band gap dataset.
+
+  Contains 4604 experimentally measured band gaps for inorganic
+  crystal structure compositions.
```

Comment:
Could you expand out this description with a brief summary of the results from the references? It might be useful to give users a sense of the results they should expect when working with this dataset